### PR TITLE
 Vue2Leaflet/Leaflet map integration

### PR DESCRIPTION
**Pull request by sdreher**
Moving away from Google maps at the moment as their pricing structure changed last June. 

screen shot 2019-02-07 at 12 14 57 pm

### CODE REVIEW

File: media/js/src/map.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    const libs = ['jquery', 'bootstrap', 'Vue', 'vue2leaflet', 'utils'];
+    requirejs(libs,
+        function($, bootstrap, Vue, vue2leaflet, utils) {
+            new Vue({
+                el: '#content',
+                data: function() {
+                    return {
+                        zoom: 5,
+                        center: vue2leaflet.L.latLng(37.0902, -95.7129),
+                        url: 'http://{s}.tile.osm.org/{z}/{x}/{y}.png',
```
Comment (nikolas) [inline media/js/src/map.js]: Is it possible to make this link `https`?

Comment (sdreher) [reply media/js/src/map.js]: Yes, you’ll the update in a subsequent pr

**Comment by nikolas** [comment]
@sdreher should this still be merged now that you're going with Google Maps after all?

**Comment by sdreher** [comment]
@nikolas - yes please. I want this in the history in case there's a shift in thinking.

**Pull request merged**